### PR TITLE
Update classes doc

### PR DESCRIPTION
**Pull request by MridulS**

### CODE REVIEW

File: networkx/classes/graph.py [hunk 1]
```diff
@@ -1039,8 +1030,13 @@ def neighbors(self, n):
 
         Notes
         -----
-        It is faster to use the idiom "in G[0]", e.g.
+        It is usually more convenient (and faster) to access the
+        adjacency dictionary as G[n]:
```
Comment (jfinkels) [inline networkx/classes/graph.py]: Needs double tickmarks around inline code.

File: networkx/classes/graph.py [hunk 2]
```diff
@@ -1039,8 +1030,13 @@ def neighbors(self, n):
...
 
+        >>> G = nx.Graph()   # or DiGraph, MultiGraph, MultiDiGraph, etc
+        >>> G.add_edge('a','b',weight=7)
```
Comment (jfinkels) [inline networkx/classes/graph.py]: Spacing.

**Comment by dschult** [comment]
can you fix the items pointed out by @jfinkels then I think this is ready...

**Comment by MridulS** [comment]
done :)

**Pull request merged**